libs/core/src/test/java/org/elasticsearch/core/GlobTests.java

```diff
@@ -177,7 +177,7 @@ interface CharPredicate {
     }
 
     private String randomAsciiString(int length) {
-        return randomAsciiString(length, ch -> ch >= ' ' && ch <= '~');
+        return randomAsciiString(length, ch -> ch >= ' ' && ch <= '~' && ch != '*');
```

Comment:
Given this is identical to the randomAsciiStringNoAsterisks introduced in https://github.com/elastic/elasticsearch/pull/133114, I think this has already been addressed?

Reply:
Indeed, thanks for noticing! I'm closing this one